german/Leichen/Ausschau/Ausschau.md

```diff
@@ -1,5 +1,7 @@
-Da! Dort dr�ben in der Ferne! Leicht �ber der Wassermarke zeichnet sich etwas ab.
-Jedes Mal, wenn die wiegenden Wogen es zu lassen sp�ht ihr gen Strand.
-Vielleicht ein St�ck Treibholz, eine verwahrloste Boje?
-Ihr k�nnt euren Kopf kaum �ber Wasser halten und schluckt jede Menge Salzwasser,
-aber ihr seit euch sicher: wenn ihr es nur bis dort hin schafft, �berlebt ihr vielleicht.
+Da! Dort drüben in der Ferne! Leicht über der Wassermarke zeichnet sich etwas ab.
+Jedes Mal, wenn die wiegenden Wogen es zulassen, späht ihr gen Strand.
+Vielleicht ein Stück Treibholz, eine verwahrloste Boje?
+Ihr könnt euren Kopf kaum über Wasser halten und schluckt jede Menge Salzwasser,
+aber ihr seit euch sicher: wenn ihr es nur bis dort hin schafft, überlebt ihr vielleicht.
+
+[Ihr schafft es bis zur Insel](https://github.com/WaterfallDeveloper/create-your-own-adventure/blob/master/german/schwimmen/insel/insel.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.